lib/thesis/view.ex

```diff
@@ -193,16 +193,21 @@ defmodule Thesis.View do
     raw """
     <script>
       ;(function () {
-        var loadThesis = function () {
+        var loadThesis = function (callback) {
```

Comment:
@GantMan this snippet dumps directly into the document (this does not compile).Can browsers handle const/let?